exla/mix.exs

```diff
@@ -52,8 +52,8 @@ defmodule EXLA.MixProject do
 
   defp deps do
     [
-      {:nx, "~> 0.4.0"},
-      # {:nx, path: "../nx"},
+      # {:nx, "~> 0.4.0"},
+      {:nx, path: "../nx"},
```

Comment:
We can still use `{:nx, "~> 0.4.0"}`, given the new functionality is a superset. :)